### ISSUE TITLE
TMP and TMPDIR variables are not passed in Singularity environment 

### ISSUE THREAD

**Issue by emi80** [nextflow-io/nextflow#496]
It seems that, by design, Singularity containers ignore the `TMPDIR` variable if defined in the calling environment or passed using `env`:
```
$ env - TMPDIR=$PWD singularity exec my_image.img env | echo "'$(grep TMPDIR)'"
```

Using [SINGULARITYENV_ prefix](https://github.com/singularityware/singularity/blob/e1d8f64168e0250d0fd90ea6a3cb43803b4f304e/etc/init#L4) should solve the issue.

**Comment by emi80** [comment]
This issue is related to #495

**Comment by pditommaso** [comment]
Included in version `0.26.0-RC2`.
